### PR TITLE
Support standing order

### PR DESCRIPTION
**Pull request by priyanshu-ap**

### CODE REVIEW

File: tpp-sample/src/RedeemStandingOrderTokenSampleTest.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                Token token =
+                    CreateStandingOrderTokenSample.CreateStandingOrderToken(payer, payeeAlias, Level.Standard);
+
+                StandingOrderSubmission submission = RedeemStandingOrderTokenSample
+                    .RedeemStandingOrderToken(payee, token.Id);
+                Assert.NotNull(submission);
+            }
+        }
+    }
+}
```
Comment (gianluca-token) [inline tpp-sample/src/RedeemStandingOrderTokenSampleTest.cs]: \n

File: tpp-sample/src/sample/RedeemStandingOrderTokenSample.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+            // Payee redeems a standing order token.
+            // Money is transferred to a payee bank account.
+            StandingOrderSubmission submission = payee.RedeemStandingOrderTokenBlocking(
+                standingOrderToken.Id);
+
+            return submission;
+        }
+    }
+}
```
Comment (gianluca-token) [inline tpp-sample/src/sample/RedeemStandingOrderTokenSample.cs]: \n

File: user-sample/src/CreateStandingOrderTokenSampleTest.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+                UserMember payer = TestUtil.CreateMemberAndLinkAccounts(tokenClient);
+                Alias payeeAlias = TestUtil.RandomAlias();
+                UserMember payee = tokenClient.CreateMemberBlocking(payeeAlias);
+                Token token =
+                    CreateStandingOrderTokenSample.CreateStandingOrderToken(payer, payeeAlias, Level.Standard);
+                Assert.NotNull(token);
+            }
+        }
+    }
+}
```
Comment (gianluca-token) [inline user-sample/src/CreateStandingOrderTokenSampleTest.cs]: \n

File: user-sample/src/RedeemStandingOrderTokenSampleTest.cs [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+                Account payeeAccount = payee.CreateTestBankAccountBlocking(1000, "EUR");
+                Token token =
+                    CreateStandingOrderTokenSample.CreateStandingOrderToken(payer, payeeAlias, Level.Standard);
+                StandingOrderSubmission submission = RedeemStandingOrderTokenSample
+                    .RedeemStandingOrderToken(payee, token.Id);
+                Assert.NotNull(submission);
+            }
+        }
+    }
+}
```
Comment (gianluca-token) [inline user-sample/src/RedeemStandingOrderTokenSampleTest.cs]: \n

File: user/src/StandingOrderTokenBuilder.cs [hunk 5]
```diff
@@ -354,7 +359,8 @@ public TokenPayload BuildPayload()
                 logger.Warn("refId is not set. A random ID will be used.");
                 payload.RefId = Util.Nonce();
             }
+
             return payload;
         }
     }
-}
+}
```
Comment (gianluca-token) [inline user/src/StandingOrderTokenBuilder.cs]: \n

File: user-sample/src/sample/CreateStandingOrderTokenSample.cs [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+            // Get the token redemption policy and resolve the token payload.
+            PrepareTokenResult result = payer.PrepareStandingOrderTokenBlocking(builder);
+            // Create the token: Default behavior is to provide the member's signature
+            // at the specified level. In other cases, it may be necessary to provide
+            // additional signatures with payer.createToken(payload, signatures).
+            Token token = payer.CreateTokenBlocking(result.TokenPayload, keyLevel);
+            return token;
+        }
+    }
+}
```
Comment (gianluca-token) [inline user-sample/src/sample/CreateStandingOrderTokenSample.cs]: \n

File: tpp-sample/src/RedeemStandingOrderTokenSampleTest.cs (hunk 1 above)
Comment (mahendra-aps) [reply tpp-sample/src/RedeemStandingOrderTokenSampleTest.cs]: done

File: tpp-sample/src/sample/RedeemStandingOrderTokenSample.cs (hunk 2 above)
Comment (mahendra-aps) [reply tpp-sample/src/sample/RedeemStandingOrderTokenSample.cs]: done

File: user-sample/src/CreateStandingOrderTokenSampleTest.cs (hunk 3 above)
Comment (mahendra-aps) [reply user-sample/src/CreateStandingOrderTokenSampleTest.cs]: done

File: user-sample/src/RedeemStandingOrderTokenSampleTest.cs (hunk 4 above)
Comment (mahendra-aps) [reply user-sample/src/RedeemStandingOrderTokenSampleTest.cs]: done

File: user/src/StandingOrderTokenBuilder.cs (hunk 5 above)
Comment (mahendra-aps) [reply user/src/StandingOrderTokenBuilder.cs]: done

File: user-sample/src/sample/CreateStandingOrderTokenSample.cs (hunk 6 above)
Comment (mahendra-aps) [reply user-sample/src/sample/CreateStandingOrderTokenSample.cs]: done

**Pull request merged**